Add glossary content (#1335)
fishtown-analytics/docs.getdbt.com: website/docs/terms/idempotent.md

---
id: idempotent
title: Idempotent
displayText: idempotent
hoverSnippet: Idempotent describes a process that is independent of previous executions of that same process.
---

Idempotent is a way to describe a particular process.  A process that is idempotent is independent of previous executions of that same process.  

> For a mathematical example, the process of adding 1 is not idempotent where as the process of multiplying by 1 is idempotent.  If I add 1 to a number and then add 1 again, I don’t get the same results. If I multiply a number by 1 and multiply by 1 again, I do get the same result.

Author: This could be easier to understand as an active sentence and using 2nd person instead of 1st.
```suggestion
For a mathematical example, adding 1 changes the results, but multiplying by 1 is idempotent. When you add 1 to a number and then add 1 again, you get different results. If you multiply a number by 1 and multiply by 1 again, you do get the same result.
```
